helpers/utility_map_helper.go

```diff
@@ -0,0 +1,12 @@
+package helpers
+
+// GetMapKeyByNumber returns a firstly key from searching by value
+func GetMapKeyByNumber(mapList map[string]int, value int) string {
```

Comment:
How about making it more explicit:

```suggestion
func GetMapKeyByNumberValue(mapList map[string]int, value int) string {
```